app.yaml

```diff
@@ -110,5 +110,8 @@ libraries:
   version: '1.6.1'
 - name: webapp2
   version: '2.5.2'
+# This environmental variable is for serving minified resources
+# when set True. It is used to run server as prod environment
```

Comment:
set True --> set to true

And let's not confuse "prod" with "minified". Maybe say: It allows minified resources to be used when running the server in the dev environment.